### PR TITLE
#32 Implemented DB layer for PaymentTransactionService and created Fu…

### PR DESCRIPTION
**Pull request by Sem4enko**
…ndsHolderEnumHandler

### CODE REVIEW

File: src/main/java/team/asd/constant/FundsHolderEnum.java [hunk 1]
```diff
@@ -6,8 +6,16 @@ public enum FundsHolderEnum {
...
 		return value;
 	}
+
+	public static FundsHolderEnum getByInt(Integer intValue) {
+		for (FundsHolderEnum v : values()) {
+			if (intValue.equals(v.value)) {
+				return v;
+			}
+		}
+		return null;
```
Comment (DenisTymchuk) [inline src/main/java/team/asd/constant/FundsHolderEnum.java]: Rewrite this method using Stream

**Comment by rdovgan** [comment]
@Sem4enko did you check all endpoints for PaymentTransaction?

File: src/test/java/team/asd/dao/TestPaymentTransactionDao.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+	public PaymentTransaction readById(Integer id) {
+		return null;
+	}
+
+	@Override
+	public void create(PaymentTransaction paymentTransaction) {
+	}
+
+	@Override
+	public PaymentTransaction update(PaymentTransaction paymentTransaction) {
```
Comment (rdovgan) [inline src/test/java/team/asd/dao/TestPaymentTransactionDao.java]: @Sem4enko this method should be void

File: src/main/java/team/asd/dao/ImplPaymentTransactionDao.java [hunk 3]
```diff
@@ -1,27 +1,32 @@
...
 
+import lombok.AllArgsConstructor;
 import org.springframework.stereotype.Repository;
 import team.asd.entity.PaymentTransaction;
+import team.asd.mapper.PaymentTransactionMapper;
 
 @Repository
-public class ImplPaymentTransactionDao implements PaymentTransactionDao{
+@AllArgsConstructor
+public class ImplPaymentTransactionDao implements PaymentTransactionDao {
```
Comment (rdovgan) [inline src/main/java/team/asd/dao/ImplPaymentTransactionDao.java]: @Sem4enko wrong class naming
Please read this: https://thomaspoignant.medium.com/java-how-to-name-interface-and-implementor-94c0fa564b87

**Review by rdovgan — APPROVED** [review]

**Pull request merged**